src/commands/init.js

```diff
@@ -115,6 +117,15 @@ async function init() {
   }
 
   await addToXcodeBuild(cwd);
+
+  const rnVersion = getReactNativeVersion(cwd);
+
+  if (semver.gte(rnVersion, '0.43.0')) {
+    await addToGradleBuild(cwd);
+  } else {
+    progress.info('Haul is not required in your build.gradle');
```

Comment:
I would probably remove this info message. There is no need to inform the user that everything is OK and he should not worry about build.gradle. What do you think? @zamotany

Reply:
Yeah, I also think this message is unnecessary.

Reply:
Will remove